administrator/components/com_admin/script.php

```diff
@@ -6238,6 +6241,8 @@ public function deleteUnexistingFiles($dryRun = false, $suppressOutput = false)
 			'/plugins/content/imagelazyload',
 			// Joomla 4.0 Beta 7
 			'/media/vendor/skipto/css',
+      // Joomla 4.0 Beta 8
+      '/administrator/templates/atum/scss/vendor/minicolors'
```

Comment:
```suggestion
			// Joomla 4.0 Beta 8
			'/administrator/templates/atum/scss/vendor/minicolors'
```
And again the spaces versus tabs thing ;-)